app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -152,5 +153,50 @@
         });
       }
     }
+
+    /*
+    * Check that the current item is infusable
+    * Only legendary or Exotic items are infusable.
+    */
+    vm.isInfusable = function isInfusable(item) {
+      // Infuse perk's id is 1270552711
+      return _.contains(item.talentPerks, 1270552711);
+    }
+
+    /*
+    * Show the icon for infusion, weapon or armor
+    */
+    vm.icon = function icon(item) {
+      var icon = "";
+      if (item.sort && item.sort === "Weapons") {
```

Comment:
Nice catch...in fact the UI is simple but maybe not intuitive as I thought. We could add some guidelines like: Click on any item here to see the result of infusion.